recipes/pyfolio/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pyfolio" %}
+{% set version = "0.8.0" %}
+{% set sha256 = "e30ee8c1932b45c4ce066cc78aee957d939e02a9427e797e2f7cd582cb0938c8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - ipython
+    - matplotlib
+    - numpy
+    - pandas
+    - pytz
+    - scipy
+    - scikit-learn
+    - seaborn
+    - pandas-datareader
+    - empyrical
+    - pymc3
+    - theano
+
+test:
+  imports:
+    - pyfolio
+    - pyfolio.tests
+
+about:
+  home: https://quantopian.github.io/pyfolio/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: 'pyfolio is a Python library for performance and risk analysis of financial portfolios'
+
+  description: |
+    pyfolio is a Python library for performance and risk analysis
+    of financial portfolios developed by Quantopian Inc. 
+    It works well with the Zipline open source backtesting library.
+
+    At the core of pyfolio is a so-called tear sheet that consists of
+    various individual plots that provide a comprehensive image of the
+    performance of a trading algorithm.
+  doc_url: https://quantopian.github.io/pyfolio/
+  dev_url: https://github.com/quantopian/pyfolio
+
+extra:
+  recipe-maintainers:
+    - alen12345
```

Comment:
May want to reach out to some of the contributors to `pyfolio` to see if they are interested in helping maintain. A few of them already help maintain packages in conda-forge. :)

Reply:
I will for sure, we are planning to move all the packages to conda-forge so some help will be much appreciated ;)